services/devices/cli/models_and_classes/destroy_agents.py

```diff
@@ -0,0 +1,129 @@
+from .ECSService import ECSService
+from .SQSService import SQSService
+from .create_agents import create_messages_list
+from .ECS_ACTIONS_ENUM import ECS_ACTIONS_ENUM
+from .S3Service import S3Service
+import re
+import time
+import os
+import logging
+import json
+from typing import Tuple
+
+
+def extract_first_and_second_word(text):
+    pattern = r'^[^-]+-([^-\s]+)-([^-\s]+)'
+    match = re.search(pattern, text)
+    if match:
+        first_word = match.group(1)
+        second_word = match.group(2)
+        return first_word, second_word
```

Comment:
You may want to add an else block here that returns "None, None" just in case the regex fails to find any patterns.